pkg/descheduler/descheduler.go

```diff
@@ -136,9 +136,11 @@ func RunDeschedulerStrategies(ctx context.Context, rs *options.DeschedulerServer
 			ignorePvcPods,
 		)
 
-		for name, f := range strategyFuncs {
-			if strategy := deschedulerPolicy.Strategies[api.StrategyName(name)]; strategy.Enabled {
+		for name, strategy := range deschedulerPolicy.Strategies {
+			if f := strategyFuncs[name]; f != nil && strategy.Enabled {
 				f(ctx, rs.Client, strategy, nodes, podEvictor)
+			} else {
+				klog.Errorf("Unknown strategy name '%s', exiting", name)
```

Comment:
This will log even when the strategy exists but is not enabled.

Reply:
Good catch, thanks